sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/ShareAsyncApiTests.java

```diff
@@ -286,30 +297,35 @@ public void deleteShareError() {
 
     @Test
     public void deleteIfExistsShare() {
```

Comment:
re-record this test too, because of the lack of `verifyComplete` in the previous test the ci pipeline is failing right now https://dev.azure.com/azure-sdk/public/_build/results?buildId=4350911&view=logs&j=09a27447-7d64-5e26-94a0-9bfb4f5f861b&t=808bff01-4728-59e5-ff90-5bb3018f39bf&l=292